types/duration.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	"time"
+)
+
+type Duration time.Duration
```

Comment:
I guess we did this just to make it usable in JSON, right? Will this have an extra code required when used with programmatic approach (building the config in go)?

Reply:
Right. Otherwise the user has to specify durations in nanoseconds instead of easy to use strings like `"10s"`. When using the programmatic way, the users has to cast `time.Duration` to `types.Duration`, like: `types.Duration(10 * time.Second)`.